Dockerfile

```
@@ -0,0 +1,38 @@
+    libgl1 \
+    libglib2.0-0 \
+    libgomp1 \
+    libsm6 \
+    libxext6 \
+    libxrender1 \
+    python3-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /app
+ARG TORCH_INDEX_URL=https://download.pytorch.org/whl/cu128
+RUN pip install --no-cache-dir --timeout 120 --retries 10 torch torchvision torchaudio --index-url ${TORCH_INDEX_URL}
```

Hadolint (2.15.1)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)